src/common/utxobased/db/NewModels/ProcessorTransaction.ts

```diff
@@ -0,0 +1,63 @@
+import { EdgeTransaction } from 'edge-core-js'
```

Comment:
This file is a dup of `src/common/utxobased/db/Models/ProcessorTransaction.ts`; we can just use the original file. 

Perhaps:

1. Mark each old file as deprecated with an "Old" suffix (i.e. `db/makeProcessorOld.ts` `db/Models/baseletsOld.ts`)
2. Rename "Models" folder to "models" (let's keep the case consistent).
3. Create the new files without the old suffixes
4. Later switch to the new files by remove the "Old" suffix from each import statement along with any implementation changes; then we can delete the old files within a single commit.

The alternative is to use a "New" suffix for new files and then _add_ the "New" suffix when updating the dependent modules, but this will require an extra rename commit from with "New" to without at a later point. However, I think this does worse of a job to communicate which files are deprecated while we progress.

Reply:
I disagree with this approach. We'd have to change any importing files to first point to the suffixed old, and then to the new file, instead of directly pointing to the new file. This feels much uncleaner to me than adding new files, then rename them and point the rest of the code at their new interfaces.